DESCRIPTION

```diff
@@ -22,7 +22,8 @@ Imports:
     R6,
     ggplot2,
     jsonlite,
-    ospsuite.utils
+    ospsuite.utils,
+    parallel
 Suggests: 

```

Comment:
any objections to adding `parallel`?  It is in validR

Reply:
Why do you need this? We are trying to parallelize stuff in .net (albeit not very successfully at the moment). If you do in in R, we will introduce potentially too much parallelism

Reply:
I just need access to the number of cores we can use (pls see comment below) to calculate the maximal number of simulations per subset.  `getAllowedCores()` is only for linux and returns `NULL` on windows. If I have access to that without this package then this can go.

Reply:
Why do we have getAllowedCores that is different than the method we have in ospsuite-r?
Typically we have this method
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/blob/develop/R/ospsuite-env.R#L73
getOSPSuiteSetting ["numberOfCores"] will return what you need. I am wondering if we should just not simply rely on it as default? (and yes this is using parallel)